drake/lcmtypes/lcmt_body_motion_data.lcm

```diff
@@ -4,13 +4,10 @@ struct lcmt_body_motion_data
 {
 	int64_t timestamp;
 
-	int32_t body_id;
+	string body_or_frame_name;
```

Comment:
the Matlab and C++ parsers disagree about body indices because of the removal of fixed joints, so we have to send body names instead